docs/GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -54,194 +24,171 @@ export COBALT_PIPELINE_NAME="Cobalt CICD Pipeline"
-
-        ![Pipeline Menu](https://user-images.githubusercontent.com/10041279/63459652-89b44f80-c41a-11e9-829a-05a6888b7673.png)
-
-    * Find and select the newly created repository from dropdown menu
-    * Import YAML by selecting 'Existing Azure Pipelines YAML file'
-        * Enter the path to the devops yaml file that lives within your newly created repo. (i.e. devops/providers/azure-devops/templates/azure-pipelines.yml)
-
-        ![Select YAML](https://user-images.githubusercontent.com/10041279/63459938-21b23900-c41b-11e9-9b9c-2dfa72e51350.png)
+        1. Select New Pipeline and then choose 'Azure Repos Git [YAML]'
 
-        > NOTE: Automatic drop-down does not always populate with yaml file options. It may be necessary to simply copy and paste the above path.
-    * Review devops pipeline YAML and only keep templates relevant to your enterprise patterns.
-        * Remove jobName configurations not relevant to your enterprise patterns. If new to Cobalt, we recommend keeping the path to the az_hello_world template as a starter template. This step can also be completed later as a code commit to your repo. Below is an example of a jobName that you may want to remove by simple deleting it.
-            ```yaml
-            configurationMatrix:
-            - jobName: az_service_single_region
-            terraformTemplatePath: 'infra/templates/az-service-single-region'
-            terraformWorkspacePrefix: 'sr'
-            environmentsToTeardownAfterRelease:
-            - 'devint'
-            ```
-    * Save and run
+            ![Pipeline Menu](https://user-images.githubusercontent.com/10041279/63459652-89b44f80-c41a-11e9-829a-05a6888b7673.png)
 
-        ![Fail Screenshot](https://user-images.githubusercontent.com/10041279/63546484-8ccd3f80-c4ef-11e9-8d9f-2f06dc725fc7.png)
+        1. Find and select the newly created repository from dropdown menu
+        1. Import YAML by selecting 'Existing Azure Pipelines YAML file'
+            * Enter the path to the devops yaml file that lives within your newly created repo. (i.e. devops/providers/azure-devops/templates/azure-pipelines.yml)
 
-        > NOTE: Azure Devops forces a run so expect this to fail. Future steps will resolve this problem.
+            ![Select YAML](https://user-images.githubusercontent.com/10041279/63459938-21b23900-c41b-11e9-9b9c-2dfa72e51350.png)
 
+            > NOTE: Automatic drop-down does not always populate with yaml file options. It may be necessary to simply copy and paste the above path.
+        1. Review devops pipeline YAML and only keep templates relevant to your enterprise patterns.
+            * Remove jobName configurations not relevant to your enterprise patterns. If new to Cobalt, we recommend keeping the path to the az_hello_world template as a starter template. This step can also be completed later as a code commit to your repo. Below is an example of a jobName that you may want to remove by simple deleting it.
```

Comment:
Please provide more context around the intention of this yaml file

Reply:
I expanded the explanation of several steps in this section.